### PR TITLE
Login

### PR DESCRIPTION
**Pull request by martinmona**
## Summary
- Completed login

## Screenshot
image

## Trello Card
https://trello.com/c/nUb1Xbks/32-login

### CODE REVIEW

File: Maquetado/login.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Book details</title>
```
Comment (manueltuero) [inline Maquetado/login.html]: :p Login

File: Maquetado/login.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      <label class="login-box-text">Email</label>
+      <input class="login-box-input" type="email">
+    </div>
+    <div class="form-group">
+      <label class="login-box-text">Password</label>
+      <input class="login-box-input" type="password">
+    </div>
+    <div class="form-buttons">
+      <button class="login-box-btn-log">Login</button>
+      <hr class="login-box-divider">
```
Comment (manueltuero) [inline Maquetado/login.html]: Maybe you can use a border-bottom here

File: Maquetado/scss/navbar.scss [hunk 3]
```diff
@@ -1,9 +1,14 @@
+@mixin border-top-cerulean {
```
Comment (manueltuero) [inline Maquetado/scss/navbar.scss]: Maybe you can pass the color of the border like argument

File: Maquetado/login.html [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    <div class="form-group">
+      <label class="login-box-text">Email</label>
+      <input class="login-box-input" type="email">
+    </div>
+    <div class="form-group">
+      <label class="login-box-text">Password</label>
+      <input class="login-box-input" type="password">
+    </div>
+    <div class="form-buttons">
+      <button class="login-box-btn-log">Login</button>
```
Comment (manueltuero) [inline Maquetado/login.html]: Rename by `btn-login` and `btn-sign-up`.

File: Maquetado/scss/login.scss [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.login-box {
+  align-items: center;
+  background-color: $wild-sand;
+  display: flex;
+  flex-direction: column;
+  height: 400px;
```
Comment (manueltuero) [inline Maquetado/scss/login.scss]: `max-height: 400px;`

File: Maquetado/scss/login.scss [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+  @include border-top-cerulean();
+  justify-content: space-between;
+  margin: 0 auto;
+  padding: 25px;
+  position: relative;
+  top: 180px;
+  width: 300px;
```
Comment (manueltuero) [inline Maquetado/scss/login.scss]: `max-width: 300px;`

File: Maquetado/scss/login.scss [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+  @include login-item(15px 0 0 0);
+}
+
+.login-box-divider {
+  border: 1px solid $silver-chalice;
+  width: 100%;
+}
+
+.form-buttons {
+  align-self: stretch;
```
Comment (manueltuero) [inline Maquetado/scss/login.scss]: Why you use stretch instead of set a height?

Comment (martinmona) [reply Maquetado/scss/login.scss]: No specific reason, is there any diference?

Comment (manueltuero) [reply Maquetado/scss/login.scss]: Yes, `stretch` stretches the children to container `max-height` and `max-width` properties. And `height` set the element height individually, this height is fixed, isn't variable.

Comment (martinmona) [reply Maquetado/scss/login.scss]: So, in this case should i put `height: 100%` besides `width: 100%` ?

File: Maquetado/login.html (hunk 2 above)
Comment (henryzarza) [reply Maquetado/login.html]: You can use `after` or `before` pseudo element

File: Maquetado/scss/login.scss (hunk 7 above)
Comment (manueltuero) [reply Maquetado/scss/login.scss]: If you want that your buttons adapt his size to container leave the stretch. If not, then replaced by a fixed height and width.

File: Maquetado/scss/login.scss [hunk 8]
```diff
@@ -0,0 +1,71 @@
+@mixin login-item($margin) {
+  @extend %login-text;
+  margin: $margin;
+  width: 100%;
+}
+
+%login-text {
+  border-radius: 5px;
+  font-size: 16px;
```
Comment (henryzarza) [inline Maquetado/scss/login.scss]: Use a variable here

File: Maquetado/scss/login.scss [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+
+.login-box {
+  align-items: center;
+  background-color: $wild-sand;
+  display: flex;
+  flex-direction: column;
+  @include border-top($cerulean);
+  justify-content: space-between;
+  margin: 0 auto;
+  max-height: 400px;
```
Comment (henryzarza) [inline Maquetado/scss/login.scss]:
```suggestion
 height: 400px;
```

File: Maquetado/scss/login.scss [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+  max-width: 300px;
```
Comment (henryzarza) [inline Maquetado/scss/login.scss]: And the `width: 100%`?

File: Maquetado/scss/login.scss [hunk 11]
```diff
@@ -0,0 +1,71 @@
...
+  padding: 25px;
+  position: relative;
+  top: 180px;
```
Comment (henryzarza) [inline Maquetado/scss/login.scss]: Why this value so exact?

File: Maquetado/scss/login.scss [hunk 12]
```diff
@@ -0,0 +1,71 @@
...
+
+.btn-login {
+  background-color: $earls-green;
+  color: $white;
+  height: 30px;
+  @include login-item(0 0 15px 0);
+}
+
+.btn-sign-up {
+  border: 2px solid  $earls-green;
```
Comment (henryzarza) [inline Maquetado/scss/login.scss]:
```suggestion
  border: 2px solid $earls-green;
```

File: Maquetado/scss/navbar.scss [hunk 13]
```diff
@@ -1,9 +1,14 @@
+@mixin border-top ($color) {
```
Comment (henryzarza) [inline Maquetado/scss/navbar.scss]: I think that not is necessary create a mixin for this

Comment (martinmona) [reply Maquetado/scss/navbar.scss]: Because i use the same in login and navbar

File: Maquetado/scss/login.scss (hunk 11 above)
Comment (martinmona) [reply Maquetado/scss/login.scss]: To make an exact copy from the invision haha

File: Maquetado/scss/login.scss (hunk 5 above)
Comment (henryzarza) [reply Maquetado/scss/login.scss]: I think that with `height` is enough or not?

Comment (manueltuero) [reply Maquetado/scss/login.scss]: Yes, it's true.
@martinmona what I told you yesterday apply only for `width` property, not for height. I'm sorry :p

**Pull request merged**